Cannot compile in Linux

I cannot compile this on Linux (Ubuntu 16.04). Not being used to C, I don't know if I should be calling `make` with some arguments, or if there is some other problem.

```
~/src/EBU-STL-tools-master$ make
gcc -D'VERSION_NUMBER="2018-01-30 16:05:15"' -std=c99 -lm -c source/ebu_shift.c
gcc -D'VERSION_NUMBER="2018-01-30 16:05:15"' -std=c99 -lm -c source/ebu.c
source/ebu.c: In function ‘BelleNuitFix’:
source/ebu.c:76:11: warning: passing argument 1 of ‘strncpy’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
   strncpy(ebu->gsi.TCF,ebu->gsi.TCP,8);
           ^
In file included from source/ebu.c:3:0:
/usr/include/string.h:128:14: note: expected ‘char * restrict’ but argument is of type ‘const unsigned char *’
 extern char *strncpy (char *__restrict __dest,
              ^
source/ebu.c:77:11: warning: passing argument 1 of ‘strncpy’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
   strncpy(ebu->gsi.TCP,"00000000",8);
           ^
In file included from source/ebu.c:3:0:
/usr/include/string.h:128:14: note: expected ‘char * restrict’ but argument is of type ‘const unsigned char *’
 extern char *strncpy (char *__restrict __dest,
              ^
source/ebu.c: In function ‘TeletextTrimControl’:
source/ebu.c:127:11: warning: passing argument 1 of ‘strncpy’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
   strncpy(ebu->gsi.TCF,ebu->gsi.TCP,8);
           ^
In file included from source/ebu.c:3:0:
/usr/include/string.h:128:14: note: expected ‘char * restrict’ but argument is of type ‘const unsigned char *’
 extern char *strncpy (char *__restrict __dest,
              ^
source/ebu.c:128:11: warning: passing argument 1 of ‘strncpy’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
   strncpy(ebu->gsi.TCP,"00000000",8);
           ^
In file included from source/ebu.c:3:0:
/usr/include/string.h:128:14: note: expected ‘char * restrict’ but argument is of type ‘const unsigned char *’
 extern char *strncpy (char *__restrict __dest,
              ^
gcc -D'VERSION_NUMBER="2018-01-30 16:05:15"' -std=c99 -lm -c source/string_utils.c
mkdir -p bin
gcc -D'VERSION_NUMBER="2018-01-30 16:05:15"' -std=c99 -lm -o bin/ebu_shift ebu_shift.o ebu.o string_utils.o
ebu.o: In function `EBUTC30to25':
ebu.c:(.text+0x102e): undefined reference to `roundf'
ebu.o: In function `EBUTC25to24':
ebu.c:(.text+0x1297): undefined reference to `roundf'
collect2: error: ld returned 1 exit status
makefile:19: recipe for target 'ebu_shift' failed
make: *** [ebu_shift] Error 1

```
After this, the bin directory has been created but is empty. The following files have been created in the main directory: 

- ebu_shift.o
- ebu.o
- string_utils.o

--- a/makefile
+++ b/makefile
@@ -5,30 +5,30 @@ all: ebu_shift ebu_dump ebu_trim ebu_30to25 srt ebu_rename
 
 ebu_30to25: ebu_30to25.o ebu.o string_utils.o
 	mkdir -p bin
-	gcc $(VERSION_NUMBER) -std=c99 -lm -o bin/ebu_30to25 ebu_30to25.o ebu.o string_utils.o
+	gcc $(VERSION_NUMBER) -std=c99 -o bin/ebu_30to25 ebu_30to25.o ebu.o string_utils.o -lm
 
 ebu_trim: ebu_trim.o ebu.o string_utils.o
 	mkdir -p bin
-	gcc $(VERSION_NUMBER) -std=c99 -lm -o bin/ebu_trim ebu_trim.o ebu.o string_utils.o
+	gcc $(VERSION_NUMBER) -std=c99 -o bin/ebu_trim ebu_trim.o ebu.o string_utils.o -lm
 
 ebu_dump: ebu_dump.o ebu.o string_utils.o
 	mkdir -p bin
-	gcc $(VERSION_NUMBER) -std=c99 -lm -o bin/ebu_dump ebu_dump.o ebu.o string_utils.o
+	gcc $(VERSION_NUMBER) -std=c99 -o bin/ebu_dump ebu_dump.o ebu.o string_utils.o -lm
 
 ebu_shift: ebu_shift.o ebu.o string_utils.o
 	mkdir -p bin
-	gcc $(VERSION_NUMBER) -std=c99 -lm -o bin/ebu_shift ebu_shift.o ebu.o string_utils.o
+	gcc $(VERSION_NUMBER) -std=c99 -o bin/ebu_shift ebu_shift.o ebu.o string_utils.o -lm
 
 srt: srt.o string_utils.o
 	mkdir -p bin
-	gcc $(VERSION_NUMBER) -std=c99 -lm -o bin/srt srt.o ebu.o string_utils.o
+	gcc $(VERSION_NUMBER) -std=c99 -o bin/srt srt.o ebu.o string_utils.o -lm
 
 ebu_rename: ebu_rename.o ebu.o string_utils.o
 	mkdir -p bin
-	gcc $(VERSION_NUMBER) -std=c99 -lm -o bin/ebu_rename ebu_rename.o ebu.o string_utils.o
+	gcc $(VERSION_NUMBER) -std=c99 -o bin/ebu_rename ebu_rename.o ebu.o string_utils.o -lm
 
 %.o: %.c
-	gcc $(VERSION_NUMBER) -std=c99 -lm -c $^
+	gcc $(VERSION_NUMBER) -std=c99 -c $^ -lm
 
 clean:
 	rm *.o

